### DISCUSSION TITLE
Sourcemap upload not working on development env

### DISCUSSION THREAD

**Discussion by rahgurung** [getsentry/sentry-react-native#2351] [Q&A] [answered]
Hi,

I have integrated Sentry RN module to the official RN template and when I do `npm run ios` to run IOS app in development mode, the source map upload doesn't happens. When I run using `npx react-native run-ios --configuration Release`, the sourcemap upload happens as expected. Any pointers why this happens?

**Comment by marandaneto** [comment, marked as answer]
@gurrrung on `debug` mode the symbolication of stack traces happens on the device, so no need of uploading source maps.
on `release`, the symbolication is on the server.

**Reply by rahgurung** [reply]
So how can i test the sentry RN in debug/development mode? Because crashes I'll see on dashboard will always be minified. I am doing crashes manually using Error() or Sentry.nativeCrash();.

**Reply by marandaneto** [reply]
Crashes during debug mode are already sent to Sentry symbolicated, there's no need to do anything, yo should see human readable stack traces.
